dm/registry.go

```diff
@@ -25,14 +25,25 @@ func init() {
 	r.MustRegisterWorkerType(WorkerDMSync, syncFactory)
 }
```

Comment:
How about move to https://github.com/hanfei1991/microcosm/blob/149c0407683c8fc5c61c556434b6c95cd6ac2c8c/executor/init.go#L12

Reply:
I think we can move caller of registry from init() of dm package to here